docs/labproject.md

```diff
@@ -187,6 +191,6 @@ If the input is valid, the service performs the following actions:
   - Configure each **submodule** with a new remote pointing to the newly created submodule repositories on the server.
 - Push the **repository contents** to the server.
   - Push each **submodule's contents** to the server.
-- Create a **team** with the _Team name_ provided by the user (or _Project name_ if unspecified).
+- Create a **team** with the _Team name_ provided by the user (or _Project name_ if unspecified), the team gets admin rights over its repositories.
```

Comment:
```suggestion
- Create a **team** with the _Team name_ provided by the user (or _Project name_ if unspecified).
The team will have admin rights over its repositories.
```

Reply:
I take it it istrue (admin rights)

Reply:
Good thing you asked.  I thought it was `admin` but it's actually just `write` permission that they get.

```suggestion
- Create a **team** with the _Team name_ provided by the user (or _Project name_ if unspecified), the team gets write permissions on its repositories.
```

If we change it to `admin` later, we should update the doc accordingly.